07_inf_for_numerical_data/inf_for_numerical_data.Rmd

```diff
@@ -61,122 +67,143 @@ Remember that you can answer this question by viewing the data in the data viewe
 by using the following command:
 
 ```{r str}
-glimpse(nc)
+glimpse(yrbss)
 ```
 
 ## Exploratory data analysis
 
-We will first start with analyzing the weight gained by mothers throughout the 
-pregnancy: `gained`.
+You will first start with analyzing the weight of the participants in kilograms:
+`weight`.
 
-Using visualization and summary statistics, describe the distribution of weight 
-gained by mothers during pregnancy. The `summary` function can be useful.
+Using visualization and summary statistics, describe the distribution of weights.
+The `summary` function can be useful.
 
 ```{r summary}
-summary(nc$gained)
+summary(yrbss$weight)
 ```
 
-1.  How many mothers are we missing weight gain data from?
+1.  How many observations are we missing weights from?
```

Comment:
Not totally opposed to this, but any chance one other variable might work, so we're not talking about weight?